package.json

```diff
@@ -74,6 +75,7 @@
     "shebang-regex": "^2.0.0",
     "ts-jest": "^25.5.1",
     "ts-node": "^3.2.0",
+    "typedoc": "0.20.36",
```

Comment:
Curious why no 🥕 ?

Reply:
There is a breaking change on typedoc from 0.20.x to 0.21.x where they drop support for TS 3.9, and only support 4.x

Reply:
I like :carrot:s